distro/pkg/deb/control

```diff
@@ -63,3 +63,11 @@ Description: parser toolkit for IETF YANG data modeling - executable tools
  during the creation of IETF YANG schemas.  The tools are not generally
  useful for normal operation where libyang primarily processes configuration
  data, not schemas.
+
+Package: libyang-data
```

Comment:
Maybe call the package `libyang-modules`?

Reply:
Sure, I'll update the PR a bit later today. Thanks!